packages/shared.net.platformapi/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@osdk/shared.net.platformapi",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/palantir/foundry-platform-typescript.git"
+  },
+  "exports": {
+    ".": {
+      "browser": "./build/browser/index.js",
+      "import": "./build/esm/index.js"
```

Comment:
I had to remove the default exports for now to get it passing with the rest of the repo, I can FLUP later to make it so everything in platform sdk repo also exports a default block